src/api/api.ts

```diff
@@ -0,0 +1,11 @@
+
+import {Query} from 'compassql/build/src/query/query';
+import {recommend} from 'compassql/build/src/recommend';
+import {Schema} from 'compassql/build/src/schema';
+
+export function fetchCompassRecommends(query: Query, schema: Schema) {
```

Comment:
fetchCompassQLResults / fetchCompassQLRecommendations ? 

(The former is shorter)